test/test_product_config.py

```diff
@@ -190,17 +190,28 @@ def test_from_config(self) -> None:
             "service_overrides": {"service1": {"host": "testhost"}},
         }
         options = Options.from_config(config)
-        assert options.develop == config["develop"]
```

Comment:
You could probably still check the main "develop" option, right?

Reply:
Hmm, I don't think that will work in the new scheme because the `develop` attribute for the `Options` class is no longer a boolean but instead a class with the various options as attributes.